core/src/com/ericc/the/game/entities/Player.java

```diff
@@ -0,0 +1,54 @@
+package com.ericc.the.game.entities;
+
+import com.badlogic.ashley.core.Entity;
+import com.badlogic.gdx.graphics.g2d.Sprite;
+import com.badlogic.gdx.math.GridPoint2;
+import com.ericc.the.game.Direction;
+import com.ericc.the.game.KeyboardControls;
+import com.ericc.the.game.Media;
+import com.ericc.the.game.components.PositionComponent;
+import com.ericc.the.game.components.RenderableComponent;
+import com.ericc.the.game.map.Map;
+
+public class Player extends Entity {
+
+    public PositionComponent pos;
+    public RenderableComponent renderable;
+
+    public Player(int x, int y) {
+        pos = new PositionComponent(x, y);
+        renderable = new RenderableComponent(new Sprite(Media.playerFront));
+        renderable.sprite.setOrigin(0, -0.5f);
+        add(pos);
+        add(renderable);
+    }
+
+    public Player(GridPoint2 pos) {
+        this(pos.x, pos.y);
+    }
+
+    public void update(KeyboardControls controls, Map map) {
```

Comment:
to trzeba totalnie stąd wywalić, nie może być tak że przekazujemy playerowi wszystko w funkcji update, a w ogóle taka funkcja update jest nie ecs-owa. Ale to będzie grubsza przekmina jak to sensownie zorganizować